actors/miner/tests/extend_sector_expiration_test.rs

```diff
@@ -728,6 +728,50 @@ fn extend_expiration2_drop_claims() {
     );
 }
 
+#[test]
+fn update_expiration_legacy_fails_on_new_sector_with_deals_fails() {
```

Comment:
You said `fails` twice. That's repetitive. And redundant. That's repetitive. And redundant.

Reply:
Oh no I said `fails` twice which is repetitive and redundant.